app/bundles/UserBundle/Resources/views/User/form.html.twig

```diff
@@ -92,6 +92,129 @@
             {{ form_rest(form) }}
 		</div>
 	</div>
+
+	<!-- Recent activity block(audit_log table) -->
+	<div class="col-md-3 bg-white height-auto">
+        {{ customContent('right.section.top', mauticTemplateVars is defined ? mauticTemplateVars : {}) }}
+			<div class="panel bg-white shd-none bdr-rds-0 bdr-w-0 mb-0">
+		<div class="panel-heading">
+			<div class="panel-title">{% trans %}mautic.core.recent.activity{% endtrans %}</div>
+		</div>
+		<div class="panel-body pt-xs">
+		{% if logs is defined and logs is not empty %}
+			<ul class="media-list media-list-feed">
+			{% for log in logs %}
+				{% if log.action == 'create' %}
+					{% set userPath = [] %}
+					{% set user_id = '1' %}
+					{% set usersEmail = log.details['email'][1] %}
```

Comment:
I'm getting 
```
500 Internal Server Error - Key "email" for array with keys "name, isAdmin, rawPermissions" does not exist.
```
on this line when editing my admin user

Reply:
strange, I don't get ant errors after editing user's information

Reply:
Perhaps for your user there is a log that have some changes on the email field. My user doesn't have that as it was created during the installation. Can you check if the log for emails exists before accessing it?

Reply:
Okay, I got you
I will check first whether this email exists. If not, I'll search user by his username